src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -211,16 +211,15 @@ public function testFindNamespaceWithSubnamespaces()
         $this->assertEquals('foo', $application->findNamespace('foo'), '->findNamespace() returns commands even if the commands are only contained in subnamespaces');
     }
 
-    /**
-     * @expectedException        \Symfony\Component\Console\Exception\CommandNotFoundException
-     * @expectedExceptionMessage The namespace "f" is ambiguous (foo, foo1).
-     */
     public function testFindAmbiguousNamespace()
     {
         $application = new Application();
         $application->add(new \BarBucCommand());
         $application->add(new \FooCommand());
         $application->add(new \Foo2Command());
+
+        $expectedMsg = "The namespace \"f\" is ambiguous.\nDid you mean one of these?\n    foo\n    foo1";
+        $this->setExpectedException('Symfony\Component\Console\Exception\CommandNotFoundException', $expectedMsg);
```

Comment:
maybe you can use `::class` there instead of the string? As the target is master this should be alright